manager/src/main/java/com/redhat/service/bridge/manager/connectors/ConnectorsApiImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.service.bridge.manager.connectors;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import com.openshift.cloud.api.connector.invoker.ApiClient;
+import com.openshift.cloud.api.connector.invoker.ApiException;
+import com.openshift.cloud.api.connector.invoker.Configuration;
+import com.openshift.cloud.api.connector.invoker.auth.HttpBearerAuth;
+import com.openshift.cloud.api.connector.models.Connector;
+
+@RequestScoped
+public class ConnectorsApiImpl implements ConnectorsApi {
+
+    @ConfigProperty(name = "mc.services.url")
+    String mcServicesBaseUrl;
+
+    @Inject
+    ConnectorAuth connectorAuth;
+
+    public Connector createConnector(Connector connector) {
+        com.openshift.cloud.api.connector.ConnectorsApi connectorsAPI = createConnectorsAPI();
+
+        try {
+            return connectorsAPI.createConnector(true, connector);
+        } catch (ApiException e) {
+            // TODO-MC error handling
+            throw new RuntimeException(e);
+        }
+
+    }
+
+    private com.openshift.cloud.api.connector.ConnectorsApi createConnectorsAPI() {
```

Comment:
Can this be `@Inject`? Feels like it would be better to try and create an instance of this bean at start-up and then fail if we cannot.